src/test/java/uk/gov/hmcts/reform/laubackend/idam/repository/IdamLogonRepositoryStartEndTimeTest.java

```diff
@@ -59,12 +59,12 @@ public void setUp() {
     }
```

Comment:
We should verify that no results are returned if startTime and endTime are not present.